engine/table/src/main/java/io/deephaven/engine/util/PickledResult.java

```diff
@@ -1,27 +0,0 @@
-/**
```

Comment:
There is a feature request to make python object columns persistable.  I can't find the ticket in DHC, so it must be in DHE.  Is this functionality we need for that feature, or would that feature be a complete redo on pickled serialization?